### DISCUSSION TITLE
Not able to view all the files in the directory, using github, when view in browser

### DISCUSSION THREAD

**Discussion by matcdac** [community/community#4512] [Projects and Issues]
Seeing following the error message :

```Sorry, we had to truncate this directory to 1,000 files. 7,855 entries were omitted from the list.```

Reference :

https://github.com/matcdac/IETF_RFCs/tree/master/RFC-docs/html

https://github.com/TurquoiseSpace/IETF_RFCs/tree/master/RFC-docs/html

Improvement Suggestion :

Perhaps GitHub can add pagination for the same view, when there are too many files / directories within a directory

**Comment by matcdac** [comment]
@candyho @rnystrom @michellemerrill @alexcnichols @tallguyjenks @bailey @abirana

Please look into it

**Comment by ClaireCJS** [comment]
I hate it so much
